baton/handler.go

```diff
@@ -14,6 +15,8 @@ func Handler() *mux.Router {
 	return m
 }
 
+var i = 0
```

Comment:
I'd consider building a stateless app a code-quality thing, see the 12 factor application (http://12factor.net/).